### DISCUSSION TITLE
Is it possible to reverse the order in which commits appear during an interactive rebase?

### DISCUSSION THREAD

**Discussion by Alizter** [magit/magit#4775] [Support] [answered]
Hi, is it possible for magit to reverse the order in which commits appear during an interactive rebase?

**Comment by tarsius** [comment, marked as answer]
No. Git prepares files and then used `$EDITOR` (in this case `emacs` with some help from `git-rebase.el`) to have the user edit them. The order has to be the same when `$EDITOR` returns and `git rebase` uses the content of the file as instructions.

Theoretically we could of course reverting the order when visiting the file and then again when returning, but that would be too complicated and not worth the effort.
